.vscode/tasks.json

```diff
@@ -1,41 +1,44 @@
 {
   // See https://go.microsoft.com/fwlink/?LinkId=733558
   // for the documentation about the tasks.json format
-  "version": "0.1.0",
+  "version": "2.0.0",
   "tasks": [
     {
-      "taskName": "Install all dependencies",
+      "label": "Install all dependencies",
+      "group": "build",
       "command": "./node_modules/.bin/lerna",
```

Comment:
Maybe we should switch to `npm` commands so this is same as what a developer does today? Also helps ensure consistency w.r.t. those scripts changing, this won't have to be updated. A good example is that this is missing the `--parallel` optimization found in the npm script.

```
command: "npm"
args: ["run", "bootstrap"]
```

Comment applies to commands below using `./node_modules/.bin/lerna` as well.